test/correctness/tuple_reduction.cpp

```diff
@@ -103,14 +102,13 @@ int main(int argc, char **argv) {
 
         // Schedule the even update steps on the gpu
         for (int i = 0; i < 10; i++) {
+            f.update(i);
```

Comment:
Here's an example of a usage that trips me up... wouldn't it be clearer here to see `f.update(i).schedule_default();`?

Reply:
schedule_default sounds like it reverts any existing schedule. I could try to implement that, but it would be different.

Reply:
Like I said, I don't care about the name and I'm not interested in bikeshedding it... pick whatever sounds right to you.

Reply:
suggestion:
- call it `unscheduled()`
- if the update stage already has any schedule, fail with an error
- if the update stage has no schedule, suppress the warning

Reply:
I was just about to type the same idea, so I guess it's a winner.

Reply:
Another benefit... one can `git grep '\.unscheduled()`.